src/main/kotlin/com/oneplanet/client/ClimatiqApiConfiguration.kt

```diff
@@ -0,0 +1,10 @@
+package com.oneplanet.client
+
+class ClimatiqApiConfiguration {
+
+    companion object {
+        const val PREFIX = "climatiq"
+        const val API_URL = "https://beta3.api.climatiq.io"
+        const val USER_TOKEN = "Bearer RWYP57SKWB48V1M5F05NGVQT1QXM"
+    }
+}
```

Comment:
If you just need to set configuration in `ClimatiqApiConfiguration`, then use [object expression](https://kotlinlang.org/docs/object-declarations.html) instead:
```suggestion
object ClimatiqApiConfiguration {
        const val PREFIX = "climatiq"
        const val API_URL = "https://beta3.api.climatiq.io"
        const val USER_TOKEN = "Bearer RWYP57SKWB48V1M5F05NGVQT1QXM"
}
```